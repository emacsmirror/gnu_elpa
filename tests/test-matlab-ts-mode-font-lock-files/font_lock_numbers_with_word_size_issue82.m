% -*- matlab-ts -*-

% See: https://github.com/acristoffers/tree-sitter-matlab/issues/82

% Doc https://www.mathworks.com/help/matlab/matlab_prog/specify-hexadecimal-and-binary-numbers.html
% 
% To specify unsigned 8-, 16-, 32-, and 64-bit integer types, use the suffixes u8, u16, u32, and u64.
% To specify signed 8-, 16-, 32-, and 64-bit integer types, use the suffixes s8, s16, s32, and s64.

a = 0xFFs8
b = 0xFFs16
c = 0xFFs32
d = 0xFFs64

e = 0xFFu8
f = 0xFFu16
g = 0xFFu32
h = 0xFFu64

i = 0b0101s8
j = 0b0101s16
k = 0b0101s32
l = 0b0101s64

m = 0b0101u8
n = 0b0101u16
o = 0b0101u32
p = 0b0101u64

