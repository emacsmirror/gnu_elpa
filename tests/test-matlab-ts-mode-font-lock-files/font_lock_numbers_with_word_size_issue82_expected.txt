c ccc ccccccccc ccc

c cccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

c ccc ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
c 
c cc ccccccc cccccccc ccc cccc cccc ccc cccccc ccccccc cccccc ccc ccc cccccccc ccc cccc cccc ccc cccc
c cc ccccccc cccccc ccc cccc cccc ccc cccccc ccccccc cccccc ccc ccc cccccccc ccc cccc cccc ccc cccc

v o nnnnnn
v o nnnnnnn
v o nnnnnnn
v o nnnnnnn

v o nnnnnn
v o nnnnnnn
v o nnnnnnn
v o nnnnnnn

V o nnnnnnnn
V o nnnnnnnnn
v o nnnnnnnnn
v o nnnnnnnnn

v o nnnnnnnn
v o nnnnnnnnn
v o nnnnnnnnn
v o nnnnnnnnn

